index.bs

```diff
@@ -1739,6 +1782,219 @@ contexts that have active documents; navigation can also cause contexts to
 become inaccessible but not yet get discarded because bfcache.
 </div>
 
+## Log ## {#module-log}
+
+The <dfn export for=modules>log</dfn> module contains functionality and events
+related to logging.
+
+### Definition ### {#module-log-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+Command = (GetCommand)
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+
+LogEvent = (
+  LogEntryAddedEvent
+)
+</pre>
+
+### Types ### {#module-log-types}
+
+#### log.LogEntry #### {#types-log-logentry}
+
+<pre class="cddl local-cddl">
+
+LogLevel = "debug" / "info" / "warning" / "error"
+
+LogEntry = {
+  type: text,
```

Comment:
```suggestion
  context: BrowsingContext,
  type: text,
```

Reply:
I think this makes sense, although of course the proposed change isn't enough to actually populate the context id. I was looking at what CDP has and it doesn't seem to have this for `Log.LogEntry`. How do existing clients/implementations work here?

Reply:
It looks like the `Log.LogEntry` event is global per connection, while [`Runtime.consoleAPICalled`](https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#event-consoleAPICalled) has an `executionContextId`.

Reply:
I remembered the reason we don't provide a context id but just a realm id is that something like a service worker has a realm but not a context. We could of course provide both, but I expect that we'll end up with a generic way of converting between them, and lots of places will take either a realm id or context id and work with both.